### ISSUE TITLE
Feature Request: Add option to allow the ability to specify the interval value for 'cpu/percent' node

### ISSUE THREAD

**Issue by MrPippin66** [NagiosEnterprises/ncpa#802]
NCPA = 2.3.1

This may become a feature request, but for check_ncpa cpu/percent reporting, I'm seeing this is reporting the long-term CPU consumption, and not the immediate CPU consumption.

Essentially, using this option:

-M cpu/percent -q warning=90,critical=95,aggregate=avg

However, despite there not being a CPU consumption issue going on, this is alerting.

Upon looking into a specific instance, the problem is due to the reporting being based on the total system counters, and not the system as it currently is.

Is there any option to have the reporting be based on the current system consumption, and not the total metrics of the systems counters.

If not, I'd like to request a feature to report the current consumption, since even after resolving a problem, this is still alerting (for a relative period of time until the system counter averages go below the threshold)

**Comment by MrPippin66** [comment]
@sawolf This has been there for almost 2 years with no evaluation. It's highly desirable to have an option to include the sample period.

We spikey loads that trigger this fairly often, and even requiring multiple checks still sees this trigger with what's really a false positive.

**Comment by sawolf** [comment]
> This has been there for almost 2 years with no evaluation

Yeah, that's not very cool of us.

> It's highly desirable to have an option to include the sample period.

Agreed that that doesn't look like it would be hard to implement. As with your more recent feature requests, I can't guarantee for you that this will get done, that's mostly up to the current NCPA developers and the development manager.

**Comment by MrPippin66** [comment]
Agreed, and understandable. I'm sure there's a very large backlog behind just getting V3 out the door
